src/DFAMin.hs

```diff
@@ -35,6 +34,26 @@ import qualified Data.List as List
 --      end;
 -- end;
 
+-- observation : Q is always subset of P
+-- let R = P \ Q. then following algorithm is the equivalent of the Hopcroft's Algorithm
+--
+-- R := {{all nonaccepting states}};
+-- Q := {{all accepting states}};
+-- while (Q is not empty) do
+--      choose a set A from Q
+--      remove A from Q and add it to R
+--      for each c in ∑ do
+--           let X be the set of states for which a transition on c leads to a state in A
+--           for each set Y in R for which X ∩ Y is nonempty and Y \ X is nonempty do
+--                replace Y in R by the greater of the two sets X ∩ Y and X \ Y
+--                add the smaller of the two sets to Q
```

Comment:
This part is clear enough, before both sets went in P, now only the bigger one does since the small one goes in Q.